xivo/mallow/tests/test_validate.py

```diff
@@ -13,7 +14,17 @@
     has_property,
     is_not,
 )
-from marshmallow import Schema, ValidationError
+
+try:
+    from marshmallow import Schema, ValidationError
+except Exception:
+
+    python_major_version = sys.version_info.major
+    if python_major_version == 2:
+        raise ImportError(
+            "Marshamallow library is incompatible with Python version %s" % sys.version
+        )
+
```

Comment:
Same comment than above

Reply:
Your suggestion did not work; I guess the change I made is more convenient.

Reply:
I got the following output when applying your suggestion:
```
______________ ERROR collecting xivo/mallow/tests/test_fields.py _______________
ImportError while importing test module '/root/src/github.com/wazo-platform/xivo-lib-python/xivo/mallow/tests/test_fields.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
xivo/mallow/tests/test_fields.py:22: in <module>
    from .. import fields
xivo/mallow/fields.py:9: in <module>
    "Marshamallow library is incompatible with Python version %s" % sys.version
E   ImportError: Marshamallow library is incompatible with Python version 2.7.16 (default, Oct 10 2019, 22:02:15) 
E   [GCC 8.3.0]
```

Reply:
Yeah and this is the expected behavior. Because pytest to *enter* in each module to collect tests. According their documentation, you can use this technique:
https://docs.pytest.org/en/6.2.x/example/pythoncollection.html#customizing-test-collection
But I commit fix